### PR TITLE
Refactor : Added URL constants 

### PR DESCRIPTION
**Pull request by srejitk**
## What

1. Replaced all the hardcoded links with constants throughout the application

## Why

1. To keep all the url's in one place ( Stored in ```url.js``` )

Addresses Issue #350

### CODE REVIEW

File: app/constants/url.js [hunk 1]
```diff
@@ -1 +1,9 @@
 export const GITHUB_URL = 'https://github.com/Real-Dev-Squad/website-my';
+export const RDS_MAIN_URL = 'http://realdevsquad.com/';
+export const RDS_WELCOME_URL = 'https://welcome.realdevsquad.com/';
+export const RDS_EVENTS_URL = 'http://realdevsquad.com/events.html';
+export const RDS_MEMBERS_URL = 'https://members.realdevsquad.com/';
+export const RDS_CRYPTO_URL = 'https://crypto.realdevsquad.com/';
+export const RDS_STATUS_URL = 'https://status.realdevsquad.com/';
+export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
```
Comment (rohan09-raj) [inline app/constants/url.js:8]: API Base Url can be imported from `config/environment.js` and then I think we can use it directly in `controllers/application.js`

File: app/constants/navbar.js [hunk 2]
```diff
@@ -1,2 +0,0 @@
-export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
```
Comment (rohan09-raj) [inline app/constants/navbar.js:1]: In `constants/signup.js` there are two more URLs present, please put them into `url.js` as well

File: app/constants/url.js (hunk 1 above)
Comment (srejitk) [reply app/constants/url.js:8]: Resolved.

File: app/constants/navbar.js (hunk 2 above)
Comment (srejitk) [reply app/constants/navbar.js:1]: Resolved

File: app/controllers/application.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
-import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/navbar';
+import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/url';
 import { action } from '@ember/object';
 import { GITHUB_URL } from '../constants/url';
+import ENV from 'website-my/config/environment'; // remove this when new flow goes live
```
Comment (rohan09-raj) [inline app/controllers/application.js:6]: Can we please remove the comment from here? It was specifically for the signup.js

File: app/controllers/application.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
 
 export default class ApplicationController extends Controller {
   @service router;
   GITHUB_URL = GITHUB_URL;
+  BASE_API_URL = ENV.BASE_API_URL;
```
Comment (rohan09-raj) [inline app/controllers/application.js:11]: Can we please use this `URL` while making a fetch call and remove the `LOGOUT_URL` from here?
image

File: app/constants/url.js [hunk 5]
```diff
@@ -1 +1,12 @@
...
```
Comment (rohan09-raj) [inline app/constants/url.js:8]: Can we please remove this URL from here as we will be use `BASE_API_URL` with template strings for this purpose?

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: app/constants/url.js (hunk 5 above)
Comment (srejitk) [reply app/constants/url.js]: This is rectified now!

**Comment by srejitk** [comment]
@satyam73 Could you please have a look into this PR

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**